Examples/BinaryBH/README.md

```diff
@@ -0,0 +1,25 @@
+# Using TwoPunctures initial data with this example
```

Comment:
This appears idiot proof as I could follow it!

Reply:
Yay, I'm glad it was straightforward :slightly_smiling_face:  In any case, I plan to add wiki page that will reiterate (and perhaps expand on this).